### PR TITLE
Fix #103 - Enable Gemini on Nightly and Set to 10% on Beta

### LINKED ISSUE #103: Enable Gemini in Nightly and set to 10% on Beta (production)

### PR DESCRIPTION
**Pull request by jumde**
fix #103

### CODE REVIEW

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Win 10 x64` using the following build:

```
Brave | 1.30.41 Chromium: 93.0.4577.51 (Official Build) nightly (64-bit)
-- | --
Revision | 762d21050e2da59930c784c09b134d0b0b148188-refs/branch-heads/4577@{#915}
OS | Windows 10 OS Version 2009 (Build 19042.1165)
```

* launched Nightly using `--variations-server-url=https://variations.bravesoftware.com/seed`
* restarted the browser once more using `--variations-server-url=https://variations.bravesoftware.com/seed`
* checked `brave://version` and ensured that `BraveRewardsGeminiNightly:Enabled` was visible
* went through `brave://rewards` and ensured that Gemini was an available option when verifying a wallet

**`Example`** | **`Example`**
---------------|----------------
geminiEnabled | gemini

**Comment by btlechowski** [comment]
Verification passed on `beta`

Brave | 1.29.67 Chromium: 92.0.4515.159 (Official Build) beta (64-bit)
-- | --
Revision | 0185b8a19c88c5dfd3e6c0da6686d799e9bc3b52-refs/branch-heads/4515@{#2052}
OS | Linux

Gemini is Enabled:
image
image


Gemini is Default state (disabled):
image
image


Verification passed on `nightly`

Brave | 1.30.19 Chromium: 92.0.4515.131 (Official Build) nightly (64-bit)
-- | --
Revision | 6b8d6c56ce21e38a72f7c4becb5abc1fa5134f29-refs/branch-heads/4515@{#1933}
OS | Linux

Default state:
image
image

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers.

**Pull request merged**
